packages/amplication-nest-logger-module/README.md

```diff
@@ -0,0 +1,15 @@
+# Amplication nestjs logger module
+
+Amplication NestJs logger module is a standards wrapper for nest-Winston that it is a wrapper for Winston logger.
+We need this library to ensure that all relevant metadata is included in every log.
```

Comment:
```suggestion
The Amplication NestJS Logger Module is our official logger module. Currently, it is a wrapper around the `nest-winston` package.

All services at Amplication must use this library to ensure consistent logging. This will enable us to trace and observe our systems reliably.
```

We need to make sure we mention technology names accurately.

`NestJs` -> `NestJS
`nest-Winston` -> `nest-winston`